<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GeneralPage</class>
 <widget class="QWidget" name="GeneralPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>463</width>
    <height>353</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>General Info</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Title:</string>
        </property>
        <property name="buddy">
         <cstring>title</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="title"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Description:</string>
        </property>
        <property name="buddy">
         <cstring>description</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QTextEdit" name="description">
        <property name="tabChangesFocus">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Autor Notes:</string>
        </property>
        <property name="buddy">
         <cstring>authorNotes</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QTextEdit" name="authorNotes">
        <property name="tabChangesFocus">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>License:</string>
        </property>
        <property name="buddy">
         <cstring>license</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Version:</string>
        </property>
        <property name="buddy">
         <cstring>version</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="version">
        <property name="text">
         <string>1.0.0</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QComboBox" name="license">
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Installation</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Target Version:</string>
        </property>
        <property name="buddy">
         <cstring>targetVersion</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLineEdit" name="targetVersion">
          <property name="text">
           <string>3.0.0</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="setLatestVersion">
          <property name="text">
           <string>Set Latest Version</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Installation Level:</string>
        </property>
        <property name="buddy">
         <cstring>installLevel</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Installation Time:</string>
        </property>
        <property name="buddy">
         <cstring>installTime</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="installTime">
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::UpDownArrows</enum>
        </property>
        <property name="correctionMode">
         <enum>QAbstractSpinBox::CorrectToNearestValue</enum>
        </property>
        <property name="value">
         <number>60</number>
        </property>
        <property name="suffix">
         <string> s</string>
        </property>
        <property name="minimum">
         <number>60</number>
        </property>
        <property name="maximum">
         <number>86400</number>
        </property>
        <property name="singleStep">
         <number>10</number>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="installLevel"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>title</tabstop>
  <tabstop>version</tabstop>
  <tabstop>description</tabstop>
  <tabstop>authorNotes</tabstop>
  <tabstop>license</tabstop>
  <tabstop>targetVersion</tabstop>
  <tabstop>setLatestVersion</tabstop>
  <tabstop>installLevel</tabstop>
  <tabstop>installTime</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
